### DISCUSSION TITLE
Thanks thanks thanks

### DISCUSSION THREAD

**Discussion by pabloavegno** [xob0t/Google-Photos-Toolkit#35] [General]
I sincerely and really want to thank you for your great application. I was looking for this kind of app for many many many years because I have a pixel one that do not take space in the storage quota, and I couldn't identify which media was staking space. I tried to do the job manually, with a web scrapper, without being able to identify them. With your application I made the job in seconds. THANKS ! Pablo from Argentina

**Comment by pabloavegno** [comment]
Dear xobOt, I need to contact you regarding one characteristic of images that do not take quota. May be with your knowledge we can do something. My mail is avegno.pablo.eeuu@gmail.com . Kind regards. Pablo

**Reply by xob0t** [reply]
Hi!
I'd prefer to keep the conversation here. If you have a question, just ask.

**Reply by pabloavegno** [reply]
No problem. Is it possible here to have a private conversation ?

**Reply by xob0t** [reply]
Its not private, but does it have to be?
